### PR TITLE
Changes:

### PR DESCRIPTION
**Pull request by NikolayAbysov**
1 GenericDao interface implemented
2 GenericService interface implemented
3 Tomcat adjusted

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/dao/impl/OrderDaoImpl.java [hunk 1]
```diff
@@ -38,4 +38,5 @@ public Order update(Order order) {
     public boolean delete(Long id) {
         return Storage.orders.removeIf(o -> o.getId().equals(id));
     }
+
```
Comment (boroda4436) [inline src/main/java/mate/academy/internetshop/dao/impl/OrderDaoImpl.java]: Redundant empty line

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
```
Comment (boroda4436) [inline src/main/webapp/WEB-INF/views/index.jsp]: Remove comments from JSP page

File: src/main/webapp/web.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <servlet>
+        <servlet-name>index</servlet-name>
+        <servlet-class>mate.academy.internetshop.controllers.IndexController</servlet-class>
+    </servlet>
+    
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+</web-app>
```
Comment (boroda4436) [inline src/main/webapp/web.xml]: the empty line at the EOF is required

**Review by boroda4436 — APPROVED** [review]

File: src/main/webapp/web.xml (hunk 3 above)
Comment (NikolayAbysov) [reply src/main/webapp/web.xml]: @boroda4436 
Hi!
Code style improved

**Pull request merged**